include/openssl/kdf.h

```diff
@@ -49,11 +50,20 @@ int EVP_KDF_derive(EVP_KDF_CTX *ctx, unsigned char *key, size_t keylen);
 # define EVP_KDF_CTRL_SET_SCRYPT_N      0x0d /* uint64_t */
 # define EVP_KDF_CTRL_SET_SCRYPT_R      0x0e /* uint32_t */
 # define EVP_KDF_CTRL_SET_SCRYPT_P      0x0f /* uint32_t */
+# define EVP_KDF_CTRL_SET_SSHKDF_XCGHASH    0x10 /* unsigned char *, size_t */
+# define EVP_KDF_CTRL_SET_SSHKDF_SESSION_ID 0x11 /* unsigned char *, size_t */
+# define EVP_KDF_CTRL_SET_SSHKDF_TYPE       0x12 /* int */
 
 # define EVP_KDF_HKDF_MODE_EXTRACT_AND_EXPAND  0
 # define EVP_KDF_HKDF_MODE_EXTRACT_ONLY        1
 # define EVP_KDF_HKDF_MODE_EXPAND_ONLY         2
 
+#define EVP_KDF_SSHKDF_TYPE_ININITAL_IV_CLI_TO_SRV 65
+#define EVP_KDF_SSHKDF_TYPE_ININITAL_IV_SRV_TO_CLI 66
```

Comment:
Is the double IN (ININITIAL) intentional or typo?

Reply:
It's a typo, INIT -> ITI, will fix today